src/dragonfs.c

```diff
@@ -979,6 +979,22 @@ uint32_t dfs_rom_addr(const char *path)
     return base_ptr+entry->data_ofs;
 }
 
+uint32_t dfs_rom_size(const char *path)
+{
+    //Skip initial slash
+    if(path[0] == '/') {
+        path++;
+    }
+    dfs_lookup_file_t *entry = lookup_file(path);
+    
+    if(!entry)
+    {
+        //File not found
+        return 0;
```

Comment:
Can you maybe change this to a negative error value like -1 to signal the error condition? I don't remember offhand whether we support 0-byte files in the filesystem, but it seems safer in general to use negative values for error values. It's not like you need more than 31 bits to represent the size anyway.

Reply:
Updated. I've also added some explicit casting between signed/unsigned when returning the value.

Let me know if there are any style concerns about bracket usage.